u8timeseries/backtesting/backtesting.py

```diff
@@ -166,3 +170,91 @@ def backtest_regression(feature_series: Iterable[TimeSeries],
         times.append(pred.end_time())  # store the N-th timestamp
 
     return TimeSeries.from_times_and_values(pd.DatetimeIndex(times), np.array(values))
+
+
+def forecasting_residuals(model: ForecastingModel, series: TimeSeries, fcast_horizon_n: int = 1,
+                          verbose: bool = True) -> TimeSeries:
+    """ A function for computing the residuals produced by a given model and time series.
+
+    This function computes the difference between the actual observations from 'series'
+    and the fitted values vector p obtained by training 'model' on 'series'.
```

Comment:
I think to get good results on Sphinx you should use these ticks: `, instead of '

Reply:
Done for that file. I created a jira taks to go over all files to update all ticks.